test/cypress/integration/getting_started_spec.ts

```diff
@@ -332,6 +306,27 @@ describe('Getting started modal', () => {
         });
     });
 
+    it('only enables one environment to be selected at a time', () => {
+        goToNextStep();
+        cy.contains('button', 'AWS').click();
+        cy.contains('button.active', 'AWS');
+
+        cy.contains('button', 'GCP').click();
+        cy.contains('button.active', 'AWS').should('not.exist');
+        cy.contains('button.active', 'GCP');
+    });
+
+    it('enables to click "next" button when an environment is being selected', () => {
```

Comment:
Maybe to be consistent with other test cases naming it'd be more accurate to have:
```suggestion
    it('should allow to click "Next" button when an environment is being selected', () => {
```
? ("allow" sounds for me better suited than "enable")

Reply:
Seems good to me,
I'll provide that inside the upcoming commit

Reply:
Done